runtime/doc/eval.txt

```diff
@@ -4071,7 +4073,7 @@ getcwd([{winnr}[, {tabnr}]])				*getcwd()*
 			getcwd(0)
```

Comment:
Ah, we had already merged a later patch, [v7.4.2273](https://github.com/vim/vim/commit/3056735ae8a366aa7fcb51872520895251858637), which reworded this help so I applied the linkification there.